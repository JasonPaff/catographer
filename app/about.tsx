import { About } from "../components/about-screen/about";

export default function AboutScreen() {
	return <About />;
}
