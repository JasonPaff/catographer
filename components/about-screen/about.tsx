import { StyleSheet, Text, View } from "react-native";
import { CatLogo } from "../shared/cat-logo/cat-logo";

export function About() {
	return (
		<View style={styles.container}>
			<Text style={styles.h1}>Catographer</Text>
			<Text style={styles.h3}>Developed By: Jason Paff</Text>
			<CatLogo height={350} marginTop={25} width={350} />
		</View>
	);
}

const styles = StyleSheet.create({
	container: {
		alignItems: "center",
		flex: 1,
		marginTop: 25,
	},
	h1: {
		color: "white",
		fontSize: 48,
		fontWeight: "bold",
	},
	h3: {
		color: "white",
		fontSize: 18,
		fontWeight: "semibold",
	},
});
