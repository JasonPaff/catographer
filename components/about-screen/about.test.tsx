import { render, screen } from "@testing-library/react-native";
import { About } from "./about";

test("<About/> should render correctly", () => {
	render(<About />).toJSON();
	expect(screen).toMatchSnapshot();
});
