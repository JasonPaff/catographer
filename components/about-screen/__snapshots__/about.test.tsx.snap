// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About/> should render correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "marginTop": 25,
    }
  }
>
  <Text
    style={
      {
        "color": "white",
        "fontSize": 48,
        "fontWeight": "bold",
      }
    }
  >
    Catographer
  </Text>
  <Text
    style={
      {
        "color": "white",
        "fontSize": 18,
        "fontWeight": "semibold",
      }
    }
  >
    Developed By: Jason Paff
  </Text>
  <Image
    alt="catographer logo"
    source={1}
    style={
      {
        "height": 350,
        "marginTop": 25,
        "objectFit": "contain",
        "width": 350,
      }
    }
    testID="cat-logo-image"
  />
</View>
`;
